core/config/initializers/00_tenant_config_schema.rb

```diff
@@ -704,6 +704,40 @@
           }
         },
       }
+    },
+    authentication: {
+      title: 'Authentication feature',
+      description: 'Authentication feature',
+      type: 'object',
+      properties: {
+        two_factor: {
+          title: 'Two factor',
+          description: 'Two factor',
+          type: 'object',
+          properties: {
+            admin_enabled: {
```

Comment:
Might it be better to call this required? as it's effectively required?
Or do we keep it this way to have the possibility to have opt in 2FA later (eg: 2fa is enabled but not required)

Reply:
That's king of what I was thinking but up to you

Reply:
@ouranos given your comment I'm unsure what solution you actually prefer, so keeping as is